lib/stripe/core_resources/customer.ex

```diff
@@ -195,4 +195,31 @@ defmodule Stripe.Customer do
     |> put_method(:delete)
     |> make_request()
   end
+
+  @doc """
+  A customer's Cash balance represents real funds.
+  Customers can add funds to their cash balance by sending a bank transfer.
+  These funds can be used for payment and can eventually be paid out to your bank account.
```

Comment:
Which is 'your bank account'? It is not very clear to me here.

Reply:
The idea is that these funds could be used to make payments(in our case payments_intents) and then we pay out this funds to the customer bank account.

We can remove that if this is confusing, I refactored the text, take a look if this sound better:
``` 
 A customer's Cash balance represents real funds.
Customers can add funds to their cash balance by sending a bank transfer.
These funds can be applied towards PaymentIntent with source “cash_balance”.
```

Reply:
I understand the concept, but I think if the customer added the funds it doesn't make sense that they are paid out to the customer's bank account. What do you think?

Reply:
Ahh I got it now, yes does not make sense to be paid out to the customer's bank account if they are the ones who added the funds. It makes sense to be used to pay something/someone. So I will remove this part. 

Since we use cash_balance for payment_intents, I removed the paid out part and described it to be used for paymentIntents.

  ```
  A customer's Cash balance represents real funds.
  Customers can add funds to their cash balance by sending a bank transfer.
  These funds can be used to reconciliate PaymentIntents with source “cash_balance”.
  The settings[reconciliation_mode] field describes whether these funds
  are applied to such PaymentIntents manually or automatically.
  ```
  
  Whay do you think?

Reply:
Sounds good :)